src/Processors/Chunk.h

```diff
@@ -343,6 +343,7 @@ class Chunk
 };
 
 using Chunks = std::vector<Chunk>;
+using ChunkList = std::list<Chunk>;
```

Comment:
does `std::forward_list` work  ?

Reply:
in somewhere, we need access the back element